src/Language/PureScript/Environment.hs

```diff
@@ -124,7 +125,7 @@ initEnvironment = Environment M.empty allPrimTypes M.empty M.empty M.empty allPr
 -- in its SCC, and everything determining X is either before it in an SCC path, or in the same SCC.
 makeTypeClassData
   :: [(Text, Maybe SourceType)]
-  -> [(Ident, SourceType)]
+  -> [(Ident, SourceType, Maybe [[Text]])]
```

Comment:
In this file, you'll notice a number of usages of `makeTypeClassData`. The usages here are always for defining compiler-supported type classes (e.g. `Row.Cons`) that don't have type class members, so changing the type signature here isn't problematic.

Reply:
This comment is irrelevant now that `makeTypeClassData` adds the VTA info.